tools/web_features/manifest.py

```diff
@@ -187,7 +187,7 @@ def write_manifest_file(path: str, web_features_map: WebFeaturesMap) -> None:
                 {
                     "version": 1,
                     "data": web_features_map
-                }, cls=WebFeatureManifestEncoder))
+                }, cls=WebFeatureManifestEncoder, sort_keys=False))
```

Comment:
Sort_keys is false by default.

> If sort_keys is true (default: False), then the output of dictionaries will be sorted by key.

https://docs.python.org/3/library/json.html#json.dumps